wordpress-mobile/gutenberg-mobile
# Add Jetpack install script

Fixes changes to Jetpack package hoisting introduced in https://github.com/Automattic/jetpack/commit/ef3305b95be354cb1e1a27327309483d354a4852

Setting up hoisting patterns in the project `.npmrc` file prevents CI from succeeding in this repo. This removes the hosting configuration from `.npmrc` when installing Jetpack on the CI. 

This also forces `npx` to use the same version of `pnpm` Jetpack is using (h/t @hypest )

To test:
- Verify CI passes 
- Verify `npm install` succeeds locally

PR submission checklist:
